server/handlers/cdn-caching.js

```diff
@@ -7,7 +7,7 @@ exports.addCacheHeadersToResult = function addCacheHeadersToResult(res, cacheKey
       res.setHeader('Vary', "Accept-Encoding");
       res.setHeader('Surrogate-Control', "private,no-cache,no-store,max-age=0");
     } else {
-      res.setHeader('Cache-Control', "public,max-age=15,s-maxage=240,stale-while-revalidate=300,stale-if-error=14400");
+      res.setHeader('Cache-Control', "public,max-age=15,s-maxage=900,stale-while-revalidate=1000,stale-if-error=14400");
```

Comment:
I think you should do this change on lines 17, 22 and 24 also.

Reply:
wasn't sure on whether we should be having this change for requests without `cachekeys`.

Reply:
isn't 900 a bit too high?

Reply:
else we should enable black knight to purge cache whenever a new deploy happens.

Reply:
@tribu has recommended the values above. BK to purge is not a bad idea, circleci if possible would be better though. 

Should we proceed merging this? 

@gja @sinujohn91 ^^^

Reply:
Approved. Merging to master, resolving conversation.